src/scss/tools/mixins/_ratings.scss

```diff
@@ -0,0 +1,33 @@
+//
+// ==========================================================================
+// Ratings variant mixin
+// Used in alert modifier classes to colorize the alert
+// ==========================================================================
+//
+
+$rating-iterations: (0, 2, 5, 6, 8);
+
+@mixin rating-fill($maxStars: 6) {
+    @for $i from 0 to $maxStars + 1 {
+        @if $i < $maxStars {
+            @each $ratingIteration in $rating-iterations {
+                @if $i == 0 and $ratingIteration == 0 {
+                    &.c-ratings--fill-00 {
+                        width: 0;
+                    }
+                }
+                @else {
+                    &.c-ratings--fill-#{$i}#{$ratingIteration} {
+                        width: (($ratingIteration * 0.1) + $i) / $maxStars * 100%;
+                    }
+                }
+
+            }
+        }
+        @else {
+            &.c-ratings--fill-#{$i}#{0} {
+                width: $i / $maxStars * 100%;
```

Comment:
If someone passes maxStars = 0 then I think width would be NaN? Not sure this would actually be an issue use case wise of the mixin?